src/headers/lz4.jl

```diff
@@ -0,0 +1,221 @@
+# Julia wrapper for header: /usr/local/include/lz4.h
+# docstrings copied from /usr/local/include/lz4.h
+
+function check_compression_error(ret::Integer, function_name::AbstractString)
+    ret == 0 && throw(LZ4Exception(function_name, "Compression failed."))
```

Comment:
Condition does appear to be set correctly:
```
/*! LZ4_compressBound() :
    Provides the maximum size that LZ4 compression may output in a "worst case" scenario (input data not compressible)
    This function is primarily useful for memory allocation purposes (destination buffer size).
    Macro LZ4_COMPRESSBOUND() is also provided for compilation-time evaluation (stack memory allocation for example).
    Note that LZ4_compress_default() compresses faster when dstCapacity is >= LZ4_compressBound(srcSize)
        inputSize  : max supported value is LZ4_MAX_INPUT_SIZE
        return : maximum output size in a "worst case" scenario
              or 0, if input size is incorrect (too large or negative)
*/
```

Reply:
I'm not sure what you mean by this. Can you explain further?

Reply:
It seemed suspicious that `0` was indicating a error condition. I just added this comment so that other reviewers wouldn't have to do the same validation.

Reply:
Ah! OK that makes sense.